creator-node/src/routes/nodeSync.js

```diff
@@ -10,7 +10,10 @@ const middlewares = require('../middlewares')
 // Dictionary tracking currently queued up syncs with debounce
 const syncQueue = {}
 const TrackSaveConcurrencyLimit = 10
+const NonTrackFileSaveConcurrencyLimit = 10
 const RehydrateIPFSConcurrencyLimit = 10
+const TrackTypes = ['track', 'copy320']
```

Comment:
this is great, you think we can store and export this from the models/tracks.js file? not sure if thats easy

Reply:
@SidSethi yeah i can attach it to the model. should it go in tracks.js? type is stored in Files so i was thinking of attaching it there

Reply:
ah sorry i meant files.js

Reply:
@SidSethi https://github.com/AudiusProject/audius-protocol/pull/664/commits/612a8a571791884d6c0792f52d69387cbfa0bbe7

Reply:
ye, perfect